expirationd/init.lua

```diff
@@ -2,6 +2,9 @@
 --
 -- @module expirationd
 
+--- Module current version.
+_VERSION = require('expirationd.version')
+
```

Comment:
```suggestion

-- ========================================================================= --
-- Expiration daemon constants
-- ========================================================================= --
--
-- {{{ Module constants
--

--- Module constants
--
-- @section Constants
--

--- Current module version in format `MAJOR.MINOR.PATCH`.
_VERSION = require('expirationd.version')

--
-- }}} Module constants
```

Could you alse move it before `Module functions` block?

```patch
--- a/expirationd/init.lua
+++ b/expirationd/init.lua
@@ -2,9 +2,6 @@
 --
 -- @module expirationd
 
---- Module current version.
-_VERSION = require('expirationd.version')
-
 -- ========================================================================= --
 -- local support functions
 -- ========================================================================= --
@@ -685,6 +682,23 @@ local function default_iterate_with(task)
     end
 end
 
+-- ========================================================================= --
+-- Expiration daemon constants
+-- ========================================================================= --
+--
+-- {{{ Module constants
+--
+
+--- Module constants
+--
+-- @section Constants
+--
+
+--- Current module version in format `MAJOR.MINOR.PATCH`.
+_VERSION = require('expirationd.version')
+
+--
+-- }}} Module constants
 
 -- ========================================================================= --
 -- Expiration daemon management functions
```

Reply:
Done